docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -26,18 +26,18 @@ zkApp programmability is not yet available on the Mina Mainnet, but zkApps can n
 
 # Tutorial 3: Deploy to a Live Network
 
-In previous tutorials, you learned how to deploy and run transactions on a local network. 
+In previous tutorials, you learned how to deploy and execute transactions on a local network. 
 
-In this tutorial, you use the `zk config` command to create a deploy alias, request tMINA funds to pay for transaction fees, and deploy a project to a live network.
+In this tutorial, you will use the `zk config` command to create the deploy alias, request tMINA funds to pay for transaction fees, and deploy zkApp to a live network.
 
 This tutorial reuses the `Square` contract that you created in [Tutorial 1: Hello World](hello-world). 
 
 ## Prerequisites
 
 This tutorial has been tested with:
 
-- [zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.17.2`
-- [o1js](https://www.npmjs.com/package/o1js) version `0.16.2`.
+- [zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.20.0`
```

Comment:
```suggestion
- [zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.21.0`
```

Reply:
We are actually on `v0.21` now.

Reply:
It is still `v0.20.0` everywhere (NPM, package.json) 
cc @ymekuria

Reply:
My mistake. There was a release tag made for v0.21 but the package.json version was not updated to `v0.21.` We will actually be releasing `v0.20.1` today.